ACID/acid_tbc.sql

```diff
@@ -11213,10 +11213,12 @@ INSERT INTO `creature_ai_scripts` (`id`,`creature_id`,`event_type`,`event_invers
 ('492201','4922','0','0','100','1025','1000','3000','180000','185000','0','0','11','7164','0','0','0','0','0','0','0','0','0','0','Guard Edward - Cast Defensive Stance on Aggro'),
 ('492202','4922','0','0','100','1025','4000','9000','7000','12000','0','0','11','11976','1','0','0','0','0','0','0','0','0','0','Guard Edward - Cast Strike'),
 ('492203','4922','2','0','100','0','15','0','0','0','0','0','25','0','0','0','1','1150','0','0','0','0','0','0','Guard Edward - Flee at 15% HP'),
+('492204','4922','0','0','100','1025','0','100','180000','185000','0','0','42','1','0','0','0','0','0','0','0','0','0','0','Guard Edward - Set Death Prevention on Aggro'),
```

Comment:
eventflags is 0 for aggro as it always happens on aggro no need for combataction nor repeat flag

Reply:
Because I don't know how to permanently set a flag on spawn :/ The documentation didn't help me in this case unfortunately